modules/002-deckhouse/hooks/set_module_image_value.go

```diff
@@ -69,9 +71,18 @@ func parseDeckhouseImage(input *go_hook.HookInput) error {
 	}
 	image := deckhouseSnapshot[0].(string)
 
+	parts := strings.SplitN(image, ":", 2)
```

Comment:
Deckhouse image can contain port. for example `air-gap-domain.com:7777/deckhouse/fe:v.1.42.0` It is important for air gap environments. We should support it. This code split string for only 2 parts.